admin/setup.py

```diff
@@ -17,4 +17,24 @@
 #
 import setuptools
 
-setuptools.setup(setup_requires=["d2to1", "pbr"], d2to1=True)
+setuptools.setup(
+    name="securedrop-admin",
```

Comment:
@legoktm didn't you want to update this to generic "securedrop team" contact info at one stage, same as the changelog entries? Given this is a new file it's as good a time as any.

Reply:
Done.